config.go

```diff
@@ -0,0 +1,97 @@
+package commando
+
+import (
+	"reflect"
+)
+
+type Config struct {
+	// Holder is the type of struct to marshal from/unmarshal info.
+	Holder interface{}
```

Comment:
consider using reflect.Type here

Reply:
It [ends up a `reflect.Type`](https://github.com/evenco/commando/pull/1/files#diff-0e426a43248661127a0c0ee115aef7a1093b635f8993b3f7ebb1dd9f05b8f249R88), this is just sugar so as a user of the lib you can pass the struct instead of needing to fiddle with `reflect`.

Reply:
I noticed, thought it'd be worth suggesting as an external change; nbd